curiouslearning/FeedTheMonsterJS
# Feat: Refactor the Level Indicator from Canvas to HTML (FM-311)

# Changes
- Created Base HTML for reusable class for creating, appending div templates and removing them.
-Created Level Field class where it handles all relating to rendering and extended using Base HTML.
-Code Clean up for Level Indicator by removing all canvas related drawings of the previous level indicator.
-Created Unit testing for both Base HTML and Level Field Component.

# How to test
- Start The FTM
-Select any game level.
-During game play, notice the level indicator on top left, it should update whenever you finish a stage in a game.

Ref: [FM-311](https://curiouslearning.atlassian.net/browse/FM-311)


[FM-311]: https://curiouslearning.atlassian.net/browse/FM-311?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
